components/eamxx/src/control/atmosphere_surface_coupling_importer.cpp

```diff
@@ -232,6 +263,9 @@ void SurfaceCouplingImporter::overwrite_iop_imports (const bool called_during_in
       const auto& info_d = col_info_d(ifield);
       const auto offset = icol*info_d.col_stride + info_d.col_offset;
       info_d.data[offset] = col_val;
+#ifdef HAVE_MOAB
+   //  TODO
```

Comment:
@iulian787 Is something supposed to happen here before we merge?

Reply:
@bartgol the case I tested (--compset F2010-SCREAMv1-MPASSI --res ne4pg2_oQU480 ) does not have iop_srf_prop flag to true, so the code does not enter this method;
I left here a mark that when we will have a case with iop_srf_prop true, MOAB will have to do something else; I am not sure yet. 
I could take it out. 

I see some test cases here
components/eamxx/cime_config/testdefs/testmods_dirs/scream/dpxx/arm97/shell_commands
components/eamxx/cime_config/testdefs/testmods_dirs/scream/dpxx/dycomsrf01/shell_commands

for which iop_srf_prop is true; 

@rljacob should I try these cases  ?  I am not even sure how to run them

Reply:
No. We'll handle the single column cases in another PR.

Reply:
Ok, thanks!